fms/modules/attention.py

```diff
@@ -267,6 +273,65 @@ def _sdpa_compute_op(
     return attn
 
 
+def _sdpa_with_sinks_op(
```

Comment:
I'd rename this to `_math_attention_with_sinks_op`, it makes it clearer it's not an optimized implementation